response_operations_ui/templates/admin/manage-user-accounts.html

```diff
@@ -130,8 +130,8 @@ <h1 class="ons-u-fs-l u-mb-xs" name="page-messages-title">Manage user accounts</
                                 "value": user.lastname,
                             },
                             {
-                                "value": '<a href="' + url_for("admin_bp.get_manage_account_groups", user_id=user.id) + '">Edit or delete user account </a>' if user.show_edit_button else "",
-                                "class": "ons-u-ta-right"
+                                "value": '<a href="' + url_for("admin_bp.get_manage_account_groups", user_id=user.id) + '" > Edit</a>' + ' | ' + '<a href="' + url_for("admin_bp.get_delete_uaa_user", user_id=user.id) + '"> Delete </a>' if user.show_edit_button else "", 
```

Comment:
Not sure the white space leading space around Edit and Delete does anything, when I change it, it doesn't move (i.e > Edit</a>)